zimbra/src/main/java/fr/openent/zimbra/service/QueueService.java

```diff
@@ -115,9 +123,17 @@ public Future<Action<T>> createAction(UUID userId, ActionType actionType, boolea
         Sql.getInstance().prepared(query.toString(), values, SqlResult.validUniqueResultHandler(handler -> {
             if (handler.isRight()) {
                 long id = handler.right().getValue().getLong(Field.ID);
-                Date createdAt = Date.from(Instant.parse(handler.right().getValue().getString(Field.CREATED_AT)));
-                Action<T> action = new Action<T>(id, userId, createdAt, actionType, approved);
-                promise.complete(action);
+                try {
+                    Date createdAt = new SimpleDateFormat(DateUtils.DATE_FORMAT_SQL).parse(handler.right().getValue().getString(Field.CREATED_AT));
+                    Action<T> action = new Action<T>(id, userId, createdAt, actionType, approved);
```

Comment:
Pas besoin de repréciser T type dans l'instanciation de la classe

```suggestion
                    Action<T> action = new Action<>(id, userId, createdAt, actionType, approved);
```